cfme/fixtures/cli.py

```diff
@@ -53,6 +53,7 @@ def fqdn_appliance(appliance):
         raise SproutException('No provision available')
     yield apps[0]
 
+    fqdn_appliance.ssh_client.close()
```

Comment:
This won't work, as ```fqdn_appliance``` isn't defined locally here and is the fixtures name. You probably want to use ```apps[0].ssh_client.close()``` instead